genref/output/md/apiserver-config.v1.md

```diff
@@ -0,0 +1,100 @@
+
+
+
+
+
+---
+title: apiserver.config.k8s.io/v1
+content_type: tool-reference
+---
+Package v1 is the v1 version of the API.
+
+## Resource Types 
+
+
+  
+- [AdmissionConfiguration](#apiserver-config-k8s-io-v1-AdmissionConfiguration)
+  
+    
+
+
+## `AdmissionConfiguration`     {#apiserver-config-k8s-io-v1-AdmissionConfiguration}
+    
+
+
+
+
+AdmissionConfiguration provides versioned configuration for admission controllers.
+
+<table class="table">
+<thead><tr><th width="30%">Field</th><th>Description</th></tr></thead>
+<tbody>
+    
+<tr><td><samp>apiVersion</samp><br/>string</td><td><samp>apiserver.config.k8s.io/v1</samp></td></tr>
+<tr><td><samp>kind</samp><br/>string</td><td><samp>AdmissionConfiguration</samp></td></tr>
+    
+
+  
+  
+<tr><td><samp>plugins</samp><br/>
```

Comment:
Is there a reason to use `<samp>` over `<code>`?

Reply:
This is because on the website, `<code>` has a gray background which is slightly darker than the shadow of the stripped table.